front/.eslintrc.json

```diff
@@ -12,6 +12,6 @@
       { "max": 75, "skipBlankLines": true, "skipComments": true }
     ],
     "complexity": ["error", 6],
-    "prettier/prettier": ["error", { "tabWidth": 4 }]
```

Comment:
you should probably still move this to a prettier file any extra configs might not be detected by vscode cause it looks at the prettierrc file for formatting

Reply:
It seems that this is not necessary according to https://khalilstemmler.com/blogs/tooling/prettier/ , and it works fine on my IDE: changing the .eslintrc updates the config of my prettier's IDE suggestions